tasks.py

```diff
@@ -25,8 +24,6 @@ async def on_invoice_paid(payment: Payment) -> None:
         and payment.extra.get("name")
         and payment.extra.get("email")
     ):
-        await api_ticket_send_ticket(
-            payment.memo,
-            payment.payment_hash,
-        )
+    
+        await set_ticket_paid(payment.payment_hash)
```

Comment:
i think you are missing `update_event_sold()` here

Reply:
`set_ticket_paid` calls `update_event_sold`